app/utils/commands.js

```diff
@@ -0,0 +1,249 @@
+const { UntilMorning } = require('./untilMorning')
+const log = require('electron-log')
+
+const allOptions = {
+  title: {
+    long: '--title',
+    short: '-T',
+    description: 'Specify title for next break (Long or Mini)',
+    withValue: true
+  },
+  text: {
+    long: '--text',
+    short: '-t',
+    description: 'Specify text for next break (Long Break only)',
+    withValue: true
+  },
+  noskip: {
+    long: '--noskip',
+    short: '-n',
+    description: 'Do not skip directly to this break (Long or Mini)',
+    withValue: false
+  },
+  duration: {
+    long: '--duration',
+    short: '-d',
+    description: 'Specify duration for pausing breaks (Pause only) [infinitely|until-morning|seconds (as number)]',
+    withValue: true
+  }
+}
+
+const allCommands = {
+  help: {
+    description: 'Show this help message'
+  },
+  version: {
+    description: 'Show current stretchly version'
+  },
+  reset: {
+    description: 'Reset breaks'
+  },
+  pause: {
+    description: 'Pause breaks',
+    options: [allOptions.duration]
+  },
+  resume: {
+    description: 'Resume from a pause'
+  },
+  toggle: {
+    description: 'Toggle breaks between resume/paused'
+  },
+  mini: {
+    description: 'Skips to and customize next Mini Break',
+    options: [allOptions.title, allOptions.noskip]
+  },
+  long: {
+    description: 'Skips to and customize next Long Break',
+    options: [allOptions.text, allOptions.title, allOptions.noskip]
+  }
+}
+
+const allExamples = [{
+  cmd: 'stretchly pause',
+  description: 'Pause breaks indefinitly'
+},
+{
+  cmd: 'stretchly pause -d 3600',
+  description: 'Pause breaks for one hour'
+},
+{
+  cmd: 'stretchly mini -T "Stretch up !"',
```

Comment:
A little nitpick: perhaps the command examples shouldn't have spaces before exclamation marks—like there's no spaces in the descriptions.

Reply:
Sorry, french habits.